FundacionParaguaya/MentorApp
# Formfields cut content

Input text fields in socio economic question pages cut the content. This is fixed with adding multiline, but there are also a small number of style related fixes:
- Text field and select placeholder doesn't have correct padding on the right (text goes over arrow for example)
- Text and select placeholders have different font weight

**To Test**
1. Go to any draft's socioeconomic screen.
2. Check that no text is cut from input fields, this needs long enough question to check.
